lib/AI Models/AI Chatbot Seasonal Advisior/api.py

```
@@ -0,0 +1,54 @@
+class SeasonalAdviceRequest(BaseModel):
+    location: str
+    crop_type: str = ""
+    hemisphere: str
+
+# 🟢 Home Route
+@app.get("/")
+def home():
+    return {"message": "Farm Advisor AI API is running!"}
+
+# 🟢 Chatbot API
+@app.post("/ask")
```

Flake8 (7.2.0)

[error] expected 2 blank lines, found 1

(E302)